website/docs/docs/build/udfs.md

```diff
@@ -234,6 +236,110 @@ import Volatility from '/snippets/_warehouse-volatility.md';
 
 <Volatility />
 
+### Defining Python UDFs
+
+You can define Python UDFs in your dbt project and use them in SQL queries, both inside and outside of dbt. Python UDFs are currently supported in Snowflake and BigQuery. 
+
+A Python UDF creates a Python function directly within your data warehouse, which you can invoke using SQL. This makes it easier to apply complex data transformations, calculations, or logic that would be difficult to define in SQL.
```

Comment:
we should add this reasoning to the FAQ on when to use a UDF instead of a macro -> https://docs.getdbt.com/faqs/Project/udfs-vs-macros